packages/jest-cli/src/reporters/getConsoleOutput.js

```diff
@@ -25,18 +25,17 @@ module.exports = (root: string, verbose: boolean, buffer: ConsoleBuffer) => {
       .map(line => CONSOLE_INDENT + line)
       .join('\n');
 
+    let typeMessage = 'console.' + type;
     if (type === 'warn') {
       message = chalk.yellow(message);
-      type = chalk.yellow(type);
+      typeMessage = chalk.yellow(typeMessage);
     } else if (type === 'error') {
       message = chalk.red(message);
-      type = chalk.red(type);
+      typeMessage = chalk.red(typeMessage);
     }
 
     return (
-      output +
-      TITLE_INDENT +
-      (verbose ? type : chalk.bold(type)) +
+      output + TITLE_INDENT + typeMessage +
```

Comment:
Actually can you try `chalk.dim()`ing the whole `typeMessage`? It works with color inside too.